SU-HSDO/suhumsci
# SHS-5149 empty heading views cards

# READY FOR REVIEW

## Summary
This PR adjusts how the card templates check for a heading tag before manually printing heading tags. This affect Vertical Card, Horizontal Card and Vertical Linked Card patterns.

## Urgency
medium

## Steps to Test
1. Visit the following links:
https://chemistry.suhumsci.loc/people/lecturers
https://jewishstudies.suhumsci.loc/academics/awards-and-opportunities
https://mrc.suhumsci.loc/videos
https://francestanford.suhumsci.loc/project-area-study/humanities-arts
2. Ensure that they are no longer producing empty heading tags (compare to: 
https://chemistry.stanford.edu/people/lecturers
https://jewishstudies.stanford.edu/academics/awards-and-opportunities
https://mrc.stanford.edu/videos
https://francestanford.stanford.edu/project-area-study/humanities-arts)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
